src/component/container/pattern_list.tsx

```diff
@@ -66,20 +71,14 @@ export class PatternListContainer extends React.Component<PatternListContainerPr
 		);
 	}
 
-	public createItemsFromFolder(folder: PatternFolder): PatternListContainerItemProps[] {
+	public createItemsFromStyleguide(styleguide: Styleguide): PatternListContainerItemProps[] {
 		const result: PatternListContainerItemProps[] = [];
-		if (folder) {
-			folder.getChildren().forEach((child: PatternFolder) => {
-				const childItem: PatternListContainerItemProps = { value: child.getName() };
-				childItem.children = this.createItemsFromFolder(child);
-				result.push(childItem);
-			});
-
-			folder.getPatterns().forEach((pattern: Pattern) => {
+		if (styleguide) {
+			styleguide.patterns.forEach((pattern: Pattern) => {
 				result.push({
-					value: pattern.getName(),
+					value: pattern.name,
```

Comment:
See above comment with get functions

Reply:
This happens a lot. If you change this please change it for all other places as well